packages/transformers/svg/src/SVGTransformer.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import nullthrows from 'nullthrows';
+import semver from 'semver';
+import collectDependencies from './dependencies';
+import parse from 'posthtml-parser';
+import render from 'posthtml-render';
+
+export default (new Transformer({
+  canReuseAST({ast}) {
+    return ast.type === 'posthtml' && semver.satisfies(ast.version, '^0.4.0');
+  },
+
+  async parse({asset}) {
+    return {
+      type: 'posthtml',
+      version: '0.4.1',
+      program: parse(await asset.getCode(), {
+        directives: [
+          {
+            name: /^\?/,
+            start: '<',
+            end: '>',
+          },
+        ],
+        lowerCaseAttributeNames: true,
```

Comment:
I think this should be false? SVG attributes are case sensitive. They are only case insensitive when embedded in an HTML file, but that should already be handled by the HTML transformer.